engine/src/main/java/org/terasology/engine/telemetry/TelemetryEmitter.java

```diff
@@ -122,11 +104,8 @@ public TelemetryEmitter build() {
             HttpClientAdapter httpClientAdapter = getDefaultAdapter(url);
             this.httpClientAdapter(httpClientAdapter);
 
-            RequestCallback  requestCallback = getDefaultRequestCallback();
-            this.requestCallback(requestCallback);
```

Comment:
why is this no longer needed?

Reply:
see PR description

Reply:
sorry, my bad, thanks for putting it into the description :green_heart: